modules/vshn-lbaas-cloudscale/outputs.tf

```
@@ -1,13 +1,13 @@
 output "api_vip" {
-  value = local.api_vip
+  value = var.lb_count > 0 ? local.api_vip[0] : ""
```

With this new approach, this should work as

```suggestion
  value = local.api_vip
```